### PR TITLE
Update Turkish.properties

### PR DESCRIPTION
**Pull request by Yssindot**

### CODE REVIEW

File: android/assets/jsons/translations/Turkish.properties [hunk 1]
```diff
@@ -763,12 +666,9 @@ Copy saved game to clipboard = Kaydedilmiş oyunu panoya kopyala
...
 Load [saveFileName] = Yükle [saveFileName]
 Delete save = Kayıtları sil
- # Requires translation!
-[saveFileName] deleted successfully. = 
- # Requires translation!
-Insufficient permissions to delete [saveFileName]. = 
- # Requires translation!
-Failed to delete [saveFileName]. = 
+[saveFileName] deleted successfully. = [saveFileName] başarıyla silindi.
+Insufficient permissions to delete [saveFileName]. = [saveileName] dosyasını silmek için yetki yok.
```
Comment (alexban011) [inline android/assets/jsons/translations/Turkish.properties:670]: you got `saveFileName` wrong here

File: android/assets/jsons/translations/Turkish.properties [hunk 2]
```diff
@@ -1351,49 +1220,38 @@ Unknown = Bilinmiyor
...
-Owned = 
- # Requires translation!
-Near [city] = 
- # Requires translation!
-Somewhere around [city] = 
- # Requires translation!
-Far away = 
+Owned = Sahipli
+Near [city] = [city] yakınında
+Somewhere around [city] = [city yakınlarında bir yerde
```
Comment (alexban011) [inline android/assets/jsons/translations/Turkish.properties:1225]: closing bracket

File: android/assets/jsons/translations/Turkish.properties [hunk 3]
```diff
@@ -1351,49 +1220,38 @@ Unknown = Bilinmiyor
...
 
- # Requires translation!
-[victoryType] Victory = 
- # Requires translation!
-Built [building] = 
- # Requires translation!
-Add all spaceship parts in capital = 
- # Requires translation!
-Destroy all players = 
+[victoryType] Victory = [victorType] Zaferi
```
Comment (alexban011) [inline android/assets/jsons/translations/Turkish.properties:1247]: `victoryType` instead of `victorType`

**Comment by yairm210** [comment]
Going to assume this is dead

**Pull request closed without merging**